composer/datasets/in_context_learning_evaluation.py

```diff
@@ -18,9 +18,10 @@
 from composer.datasets.utils import stop_sequences_criteria
 from composer.utils import MissingConditionalImportError, dist, get_file
 
+from datasets import Dataset as HFDataset  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
this can't be at the top level because datasets is an option dependency

Reply:
https://github.com/mosaicml/composer/blob/375ea0cc3f8e0f5fadf77825f06f2909f87e9fe3/composer/datasets/in_context_learning_evaluation.py#L294-L302

Reply:
and https://github.com/mosaicml/composer/blob/375ea0cc3f8e0f5fadf77825f06f2909f87e9fe3/composer/datasets/in_context_learning_evaluation.py#L21-L23

Reply:
Fixed the import